main.go

```diff
@@ -11,12 +11,23 @@ import (
 
 	"git.jojii.de/jojii/zimserver/zim"
 	log "github.com/sirupsen/logrus"
+	"github.com/pelletier/go-toml"
 )
 
 func main() {
 	setupLogger()
 
-	libPath := "./library"
+	config, err := toml.LoadFile("config.toml")
+
+	// If the configuration file does not exist, return an error
+	if err != nil {
+		log.Error(err)
```

Comment:
`log.Error(err)` doesn't really stop the program, it just prints out the error and continues. Maybe a `return` or even better a default config file generation at this point would make more sense?

Reply:
En effet, je ne sais pas pourquoi je n'ai pas ajouté une configuration par défaut...
C'est tellement logique de faire ça comme ça.
Please check 31436ea and a4b146c. :)

Reply:
Indeed, I don't know why I didn't add a default configuration...
It's so logical to do it like that in fact.
Please check 31436ea and a4b146c. :)